### PR TITLE
CSS2DRenderer: add z respecting

### PR DESCRIPTION
**Pull request by yomotsu**
This PR is adding z respecting for CSS2DRenderer with similar way to IE11 support in CSS3DRenderer.

Current CSS2DRenderer does not respect depth.
Here is the problem: Label "Moon" is always front
https://jsfiddle.net/7w462nk9/

I also added an example for CSS2DRenderer which is missing in the examples.
https://rawgit.com/yomotsu/three.js/css2d-renderer-respect-z/examples/index.html?q=css2d#css2d_label

### CODE REVIEW

File: examples/css2d_label.html [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+			function animate() {
+
+				requestAnimationFrame( animate );
+
+				var elapsed = clock.getElapsedTime();
+
+				moon.position.set( Math.sin( elapsed ) * 5, 0, Math.cos( elapsed ) * 5 );
+
+				renderer.render( scene, camera );
+				renderer2.render( scene, camera );
```
Comment (Mugen87) [inline examples/css2d_label.html]: Can you please rename `renderer2` to `labelRenderer`?

File: examples/js/renderers/CSS2DRenderer.js [hunk 2]
```diff
@@ -95,6 +105,57 @@ THREE.CSS2DRenderer = function () {
...
+
+		scene.traverse( function ( object ) {
+
+			if ( object instanceof THREE.CSS2DObject ) result.push( object );
+
+		} );
+
+		return result;
+
+	}
```
Comment (Mugen87) [inline examples/js/renderers/CSS2DRenderer.js]: Missing comma.

**Review by Mugen87 — CHANGES_REQUESTED** [review]

**Comment by yomotsu** [comment]
@Mugen87 Thanks for your review. I changed them.

**Review by Mugen87 — APPROVED** [review]

**Comment by mrdoob** [comment]
Thanks!

**Pull request merged**